<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>514</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="infoWidget" native="true">
      <layout class="QHBoxLayout" name="infoLayout">
       <item>
        <layout class="QHBoxLayout" name="figureInfoLayout">
         <item>
          <widget class="QLabel" name="figureInfoLabel">
           <property name="text">
            <string>No figure selected.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="colorLabel">
           <property name="text">
            <string>Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="colorButton">
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string>background-color: black;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fillColorLabel">
           <property name="text">
            <string>Fill color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fillColorButton">
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string>background-color: black;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearColorButton">
           <property name="text">
            <string>Clear color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="thicknessLabel">
           <property name="text">
            <string>Thickness:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="thicknessSpinBox">
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="selectImgButton">
         <property name="text">
          <string>Select Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeImgButton">
         <property name="text">
          <string>Remove Image</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="toolbarLayout" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: lightgray;</string>
        </property>
        <layout class="QVBoxLayout" name="toolLayout" stretch="0,0,0,0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QPushButton" name="mousePointerButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/toolbar/assets/mouse.png</normaloff>:/toolbar/assets/mouse.png</iconset>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QPushButton" name="lineButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/toolbar/assets/line.png</normaloff>:/toolbar/assets/line.png</iconset>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QPushButton" name="polygonButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/toolbar/assets/polygon.png</normaloff>:/toolbar/assets/polygon.png</iconset>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QPushButton" name="circleButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/toolbar/assets/circle.png</normaloff>:/toolbar/assets/circle.png</iconset>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QPushButton" name="rectangleButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/toolbar/assets/rectangle.png</normaloff>:/toolbar/assets/rectangle.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cliperkaButton">
           <property name="text">
            <string>cliperka</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QPushButton" name="bucketButton">
           <property name="text">
            <string>Bucket</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Canvas" name="canvas" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionAnti_aliasing"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAnti_aliasing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Anti-aliasing</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
